userspace/libsinsp/parsers.h

```diff
@@ -146,6 +146,7 @@ class sinsp_parser
 	void parse_getsockopt_exit(sinsp_evt *evt);
 
 	inline void add_socket(sinsp_evt* evt, int64_t fd, uint32_t domain, uint32_t type, uint32_t protocol);
+	inline void infer_sendto_fdinfo(sinsp_evt* evt);
```

Comment:
`const` is missing here. is that on purpose?
nit: move `*` adjacent to the var name.

Reply:
Yes, that's intentional -- the `const` really ought not be here since consumers of the API don't care whether or not the implementation chooses not to modify the local copy of the pointer.

I'll move the `*`, but I'll also note that the immediately preceding method has it in the same place that I have it, *cough* clang-format *cough* :P  It also makes it somewhat inconsistent with the placement in the implementation file (with the `const` between it and the name).